docs/docs/adrs/adr-005-cryptographic-equivocation-verification.md

```diff
@@ -73,17 +74,73 @@ the light client isn’t expired.
+referred to as [`infraction_block_header`](https://github.com/cosmos/interchain-security/blob/20b0e35a6d45111bd7bfeb6845417ba752c67c60/proto/interchain_security/ccv/provider/v1/tx.proto#L77). 
+The latter will provide the malicious validator's public key and the chain ID required to verify the signature of the votes contained in the evidence.
+ 
+Note that double signing evidence will be verified by using the same conditions than in the CometBFT 
+[`verify(evidence types.Evidence)`](https://github.com/cometbft/cometbft/blob/2af25aea6cfe6ac4ddac40ceddfb8c8eee17d0e6/evidence/verify.go#L19) 
+method, with the exception that its age won't be checked. 
+This is primarily because, for the first stage of this feature, 
+the goal is to jail validators for double signing only. Since the jail time doesn't depend on the evidence's age, 
+it can be ignored. More technical details can be found in the ["Current limitations"](#current-limitations) section below. 
+  
+  
+Upon a successful equivocation verification, the misbehaving validator will be jailed for the maximum time 
+(see [DoubleSignJailEndTime](https://github.com/cosmos/cosmos-sdk/blob/cd272d525ae2cf244c53433b6eb1e835783d7531/x/evidence/types/params.go) 
+in the SDK evidence module).
+
+
+### Current limitations:
 
-- We cannot derive an infraction height from the evidence, so it is only possible to tombstone validators, not actually slash them.
+- We cannot derive an infraction height from the evidence, so it is only possible to jail validators, not actually slash them.
 To explain the technical reasons behind this limitation, let's recap the initial consumer initiated slashing logic.
-In a nutshell, consumer heights are mapped to provider heights through VSCPackets, namely through the so called vscIDs.
-When an infraction occurs on the consumer, a SlashPacket containing the vscID obtained from mapping the consumer infraction height
-is sent to the provider. Upon receiving the packet, the provider maps the consumer infraction height to a local infraction height,
-which is used to slash the misbehaving validator. In the context of untrusted consumer chains, all their states, including vscIDs,
-could be corrupted and therefore cannot be used for slashing purposes.
+ In a nutshell, consumer heights are mapped to provider heights through VSCPackets, namely through the so called vscIDs.
+ When an infraction occurs on the consumer, a SlashPacket containing the vscID obtained from mapping the consumer infraction height
+ is sent to the provider. Upon receiving the packet, the provider maps the consumer infraction height to a local infraction height,
+ which is used to slash the misbehaving validator. In the context of untrusted consumer chains, all their states, including vscIDs,
+ could be corrupted and therefore cannot be used for slashing purposes.
+
+- For the same reasons explained above, the age of a consumer double signing evidence can't be verified, 
```

Comment:
Does this apply to only double signing evidence? Are we verifying the age of light client attack evidence?